src/EditorFeatures/Core/EditorFeatures.csproj

```diff
@@ -105,6 +105,8 @@
     <InternalsVisibleToTypeScript Include="Microsoft.VisualStudio.LanguageServices.TypeScript" />
     <InternalsVisibleToTypeScript Include="Microsoft.Test.Apex.VisualStudio" />
     <InternalsVisibleToTypeScript Include="Roslyn.Services.Editor.TypeScript.UnitTests" />
+    <InternalsVisibleToRemoteLS Include="Microsoft.VisualStudio.LanguageServices.Remote" /> 
+    <InternalsVisibleToRemoteLS Include="Microsoft.VisualStudio.LanguageServices.Remote.CSharp" /> 
```

Comment:
Since every one of these makes it more difficult for users to reason about the impact of seemingly non-breaking changes to the "internal" API, I have a few questions:

❓ What are these two assemblies?
❓ Why are these two assemblies not part of this repository?
❓ What step(s) are being take to ensure "breaking changes" to internal APIs are detected early (prior to release)?

Reply:
This is for early experimentation and prototyping of some ideas. We need to figure out the right long term strategy if these ideas have legs.